Jamulus.pro

```diff
@@ -1078,4 +1078,8 @@ contains(CONFIG, "disable_version_check") {
     DEFINES += DISABLE_VERSION_CHECK
 }
 
+contains(CONFIG, "nosound") {
+    message(Restricting build to server-only due to CONFIG+=nosound.)
+    DEFINES += SERVER_ONLY
+}
```

Comment:
Since the `nosound` option is only otherwise used within `else:unix`, I would move this section up to line 343 or 356, and make one of them an `else`. Then it doesn't affect any other platforms.

Reply:
> (For clarity, it is WITH_SOUND that is split all over Jamulus.pro, not the nosound config option.)

Yes, this is where I got confused. Moved it.